release_build_files/readme.md

```diff
@@ -569,6 +569,9 @@ code.
 ## Release Notes
 ### Next Release
 -   Changes
+    -   All Products (Android): Fixed a data race that could manifest as null
```

Comment:
Can you change this to "General (Android):"?

Reply:
Done.